src/store/saga/UserInfoForm/UserInfoForm.ts

```diff
@@ -0,0 +1,31 @@
+import { ADD_USER_INFO } from "store/types";
+import { call, takeLatest, put } from "redux-saga/effects";
+import axios from "axios";
+import { addApiDataFromRedux } from "store/actions/addApiDataFromRedux";
+import { setReduxKey } from "store/actions/App";
+interface ResponseGenerator {
+  data: any;
+}
+let some: any = [];
```

Comment:
do not make global variable as we avoid global variables , also what kind of name is some I cant understand its purpose by its name.

Reply:
alright